packages/rke2-cilium/charts/requirements.yaml

```diff
@@ -0,0 +1,8 @@
+dependencies: # A list of the chart requirements (optional)
+  - name: cilium
+    repository: https://helm.cilium.io
```

Comment:
I think we need to mirror the subcharts, rather than leaving them as an external reference.

Reply:
It's doing it for us:
https://github.com/rancher/rke2-charts/blob/b40561ff50a916ea201798c14904cd7a233c0408/scripts/prepare-subcharts#L81

And documented as such:
https://github.com/rancher/rke2-charts/blob/b40561ff50a916ea201798c14904cd7a233c0408/scripts/prepare-subcharts#L14

Although I am not completly convinced this is not a happy coincidence.